dataactcore/models/jobTrackerInterface.py

```diff
@@ -241,16 +240,20 @@ def getNumberOfRowsById(self,jobId):
         """ Get number of rows in file for job matching ID """
         return self.getJobById(jobId).number_of_rows
 
+    def getNumberOfValidRowsById(self, jobId):
+        """Get number of file's rows that passed validations."""
+        return self.getJobById(jobId).number_of_rows_valid
```

Comment:
Good question re: `NULL` handling. So, based on our Slack convo, `NULL` is expected in this field if a job is not yet complete or if it's not a file validation job. Thus, it does make sense for the calling function to make the 0 vs `NULL` interpretation.

As of right now, the only caller is the validation unit tests, which use `getNumberOfValidRowsById` to check the expected number of valid rows in the test files. If, for some reason, that call gets a `NULL` back, the test would fail (which is what we'd want).